src/mozanalysis/types.py

```diff
@@ -19,3 +19,8 @@ class Uplift(str, Enum):
 EstimatesByBranch = dict[BranchLabel, Estimates]
 
 CompareBranchesOutput = dict[ComparativeOption, EstimatesByBranch]
+
+
+class ExperimentalUnit(str, Enum):
+    CLIENT = "client_id"
+    GROUP = "profile_group_id"
```

Comment:
What do you think about calling this `PROFILE_GROUP`?

Reply:
I think that would add some clarity, especially since that's what the ID was named.

Reply:
Done